mmdet3d/models/decode_heads/decode_head.py

```diff
@@ -113,8 +111,6 @@ def loss(self, inputs: List[Tensor], batch_data_samples: SampleList,
         Args:
             inputs (list[torch.Tensor]): List of multi-level point features.
             img_metas (list[dict]): Meta information of each sample.
```

Comment:
Please also modify this argument description to fit the refactored function (batch_data_samples)